src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1739,19 +1740,22 @@
   ([{:keys [certname producer_timestamp] :as report} :- reports/report-wireformat-schema
    received-timestamp :- pls/Timestamp
    db conn-status]
-   (add-report! report received-timestamp db conn-status true))
+   (add-report! report received-timestamp db conn-status true {}))
```

Comment:
we'll need to adjust how this adds the extra arity to the `add-report!` func. We'll still need to support calls with only 5 args. I think if we remove the defaulting for the `blocklist-config` add another block like the one below in addition to the final 6 arg one you added it should work. 
```
([{:keys [certname producer_timestamp] :as report} :- reports/report-wireformat-schema
    received-timestamp :- pls/Timestamp
    db conn-status update-latest-report?]
   (add-report! report received-timestamp db conn-status update-latest-report? {}))
 ```